### PR TITLE
Lerna prepack issue fixes - CS-28539

### PR DESCRIPTION
**Pull request by antonyagustine**
Lerna prepack issue fixes - CS-28539
- Updated typescript version
- Updated ts config
- Few type fixes
- Code smell fixes

### CODE REVIEW

File: packages/contentstack-bootstrap/src/commands/cm/bootstrap.ts [hunk 1]
```diff
@@ -58,44 +58,53 @@ export default class BootstrapCommand extends Command {
 
     try {
       if (!this.authToken) {
-        this.error(messageHandler.parse('CLI_BOOTSTRAP_LOGIN_FAILED'), { exit: 2, suggestions: ['https://www.contentstack.com/docs/developers/cli/authentication/'] })
+        this.error(messageHandler.parse("CLI_BOOTSTRAP_LOGIN_FAILED"), {
+          exit: 2,
```
[the remark this answers is not in the document]
Comment (shafeeqd959) [reply packages/contentstack-bootstrap/src/commands/cm/bootstrap.ts:62]: @antonyagustine

[the remark this answers is not in the document]
Comment (antonyagustine) [reply packages/contentstack-bootstrap/src/commands/cm/bootstrap.ts:62]: > here, can we suggest the command for login as we are doing this in other places. ?

We already showing messge to user as follows _messageHandler.parse('CLI_BOOTSTRAP_LOGIN_FAILED')_  Which basically refers as _You need to login, first. See: auth:login --help_. Would you like to change this message or show new log before exit.

[the remark this answers is not in the document]
Comment (antonyagustine) [reply packages/contentstack-bootstrap/src/commands/cm/bootstrap.ts:62]: @shafeeqd959 @netrajpatel Can you guys clarify me with above query..?

[the remark this answers is not in the document]
Comment (shafeeqd959) [reply packages/contentstack-bootstrap/src/commands/cm/bootstrap.ts:62]: this is good enough

**Pull request merged**